src/test/java/de/unistuttgart/iste/gits/flashcard_service/api/mutation/MutationDeleteFlashcardSetTest.java

```diff
@@ -0,0 +1,62 @@
+package de.unistuttgart.iste.gits.flashcard_service.api.mutation;
+
+import de.unistuttgart.iste.gits.common.testutil.GraphQlApiTest;
+import de.unistuttgart.iste.gits.common.testutil.TablesToDelete;
+
+import de.unistuttgart.iste.gits.flashcard_service.persistence.dao.FlashcardSetEntity;
+import de.unistuttgart.iste.gits.flashcard_service.persistence.repository.FlashcardRepository;
+import de.unistuttgart.iste.gits.flashcard_service.persistence.repository.FlashcardSetRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.graphql.test.tester.GraphQlTester;
+import org.springframework.test.annotation.Commit;
+import org.springframework.transaction.annotation.Transactional;
+import java.util.Optional;
+import java.util.UUID;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@GraphQlApiTest
+@TablesToDelete({"flashcard_side", "flashcard", "flashcard_set"})
+public class MutationDeleteFlashcardSetTest {
+
+    @Autowired
+    private FlashcardSetRepository flashcardSetRepository;
+
+    @Test
+    @Transactional
+    @Commit
+    void testDeleteFlashcard(GraphQlTester graphQlTester) {
+        // Create and save the flashcard to be deleted
+        FlashcardSetEntity flashcardSet = new FlashcardSetEntity();
+        flashcardSet.setAssessmentId(UUID.randomUUID());
+
```

Comment:
Maybe add a flashcard to this set to check if it is deleted as well (assert that no flashcards exists at the end)